paasta_tools/utils.py

```diff
@@ -257,13 +257,13 @@ def format_docker_parameters(self, with_labels=True):
         parameters.extend(self.get_cap_add())
         return parameters
 
-    def get_disk(self):
+    def get_disk(self, default=1024):
```

Comment:
i want to be able to provide a sensible default in the caller. right now the default is hardcoded in this method, it's simply a matter of convenience to expose it as optional arg.